tests/validatingadmissionpolicy/noderestrictions.go

```diff
@@ -193,4 +206,69 @@ var _ = Describe("[sig-compute] virt-handler node restrictions via validatingAdm
 			Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("%s should not fail on node specific node restriction", description))
 		}
 	})
+
+	Context("patching another node", func() {
+		BeforeEach(func() {
+			nodesList := libnode.GetAllSchedulableNodes(virtClient)
+			Expect(nodesList.Items).ToNot(BeEmpty())
+			Expect(nodesList.Items).ToNot(HaveLen(1))
+			for _, node := range nodesList.Items {
+				if nodeName != node.Name {
+					anotherNode = node.Name
+					break
+				}
+			}
+
+			prepareNode(anotherNode)
+		})
+
+		AfterEach(func() {
+			cleanup(anotherNode)
+		})
+
+		It("rejects kubevirt related patches", func() {
+			patchSetList := map[string]*patch.PatchSet{
+				"kubevirt.io label addition":      patch.New(patch.WithAdd("/metadata/labels/kubevirt.io~1newAllowedLabel", "value")),
```

Comment:
Nit:
```suggestion
				"kubevirt.io label addition":      patch.New(patch.WithAdd(allowedLabelPath + "new", "value")),
```
Same below with annotation